Assets/LeapMotionModules/GraphicRenderer/Scripts/Utility/AttachedObjectHandler.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+using Leap.Unity.Query;
+
+namespace Leap.Unity.GraphicalRenderer {
+
+  public static class SerializedObjectExtensions {
+
+    /// <summary>
+    /// Returns whether or not the given objects holds a serialized reference to a
+    /// specific object.  Simply serializes the object and steps through all properties,
+    /// so it is not a cheap operation.
+    /// </summary>
+    public static bool DoesReference(this Object obj, Object reference) {
+      SerializedObject sobj = new SerializedObject(obj);
```

Comment:
SerializedObject requires UnityEditor, so currently this doesn't compile for builds